test/evergreen.yml

```diff
@@ -6932,3 +6974,15 @@ buildvariants:
     - name: format-abort-recovery-stress-test
     - name: ".cppsuite-stress-test"
       batchtime: 720 # 12 hours
+
+- name: amazon2023-disagg-stress
+  display_name: "(ARMV9) Amazon 2023 Disagg Stress tests"
+  run_on:
+  - amazon2023-arm64-latest-large-m8g
+  expansions:
+    ENABLE_TCMALLOC: 1
+    num_jobs: $(echo "`grep -c ^processor /proc/cpuinfo`" | bc)
+  tasks:
+    - name: compile
+    - name: ".stress-test-disagg"
+      batchtime: 1440 # once a day
```

Comment:
I thought we wanted to run this once/twice a week?

Reply:
Hmm, in that case I really wish we could exclude this from the BBUI tool, then we can at our convenience look at these.
Should we investigate if there is a way?

Reply:
I am still concerned about this, but if you feel confident that it will not inconvenience triagers and we will be on top of these issues then I am okay.